### PR TITLE
Remove margin from body tag, increase Styleguidist font size

### PR DESCRIPTION
**Pull request by swese44**
The only issue i see in MC is margin on the body tag, this fixes it. Also increases Styleguidist font size -- it's importing the base 10px size from the html tag, since it no longer gets the 1.5rem base font size.

## Pull request checklist

* [x] Component `README.md` file is up-to-date.
* [x] Component is unit tested.

### CODE REVIEW

File: src/css/base.css [hunk 1]
```diff
@@ -1,6 +1,12 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 @import 'src/css/variables/colors.css';
 
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
```
Comment (chrismohr) [inline src/css/base.css]: Is there a reason not to put these on the AppContainer?
https://github.com/Microsoft/YamUI/blob/master/src/components/AppContainer/AppContainer.styles.ts#L21

Comment (swese44) [reply src/css/base.css]: i guess just to avoid more global CSS for SP, but this wouldn't affect their page. should i move the rules over to AppContainer?

Comment (chrismohr) [reply src/css/base.css]: Oh.. I see.  If it's specifically for the modern web client.  Maybe best would be to add the css there?

Comment (swese44) [reply src/css/base.css]: updated

**Pull request merged**